importers/util/common.go

```diff
@@ -0,0 +1,218 @@
+// Package util provides common code for all the importers
```

Comment:
Since we're using GenericData for almost everything, perhaps we move the struct in here?